server/historian/packages/historian-base/src/services/restGitService.ts

```diff
@@ -417,4 +439,8 @@ export class RestGitService {
             return fetch();
         }
     }
+
+    private getSummaryCacheKey() {
+        return `${this.tenantId}:${this.documentId}:summary`;
+    }
 }
```

Comment:
Can we pass the suffix (= summary) as a param? And use "container" instead of summary? This way we will be able to cache both summary and channel in future.

Reply:
Done :)